RELEASE-NOTES.txt

```diff
@@ -1,5 +1,6 @@
 Unreleased
 ==========
+* Upgraded Playground dependencies, resulting in faster offline site creation #1594
```

Comment:
It sounds like improving offline site creation was the main change/goal, but migrating from deprecated wp-now to Playground CLI was the primary one. Should we update the message to account for that?

Reply:
Updated: https://github.com/Automattic/studio/pull/1669/commits/e591473a7c1ee302652eeeb71e647eddddd53222

Another alternative is to add different lines for each purpose and tag the same PR at the end.